python-sdk/src/astro/sql/table.py

```diff
@@ -37,6 +40,13 @@ class Table:
     If no name is given, it auto-generates a name for the Table and considers it temporary.
 
     Temporary tables are prefixed with the prefix TEMP_PREFIX.
+
+    :param conn_id: The Airflow connection id. This will be used to identify the right database type at the runtime
+    :param name: The name of the database table. If name not provided then it would create a temporary name
+    :param metadata: A metadata object which will have database or schema name
+    :param columns: Table columns names.
+        If columns values provided then it would create table having provided columns names
+        otherwise it autodetect columns name from dataset
```

Comment:
```suggestion
```
I believe this information shouldn't be here since to use columns to create a table is a behaviour of an operator (`load_file`) and not of the `table` definition itself. In other operators `columns` is not used to modify the table schema (e.g. `append` and `merge`).